gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/integration/model/IntegrationApi.java

```diff
@@ -47,6 +49,17 @@ public enum PageType {
         SWAGGER,
     }
 
+    public record Metadata(String name, String value, Format format) {
+        public enum Format {
```

Comment:
I'm wondering if we could use the existing enum `io.gravitee.apim.core.metadata.model.Metadata.MetadataFormat` 🤔